src/components/Questions/Questions.js

```diff
@@ -0,0 +1,91 @@
+import { Button } from "@material-ui/core";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import ErrorMessage from "./ErrorMessage";
+import "./Questions.css";
+
+export const Questions = ({
+  currQues,
+  setCurrQues,
+  questions,
+  options,
+  correct,
+  setScores,
+  scores,
+  setQuestions,
+}) => {
+  const [selected, setSelected] = useState();
+  const [error, setError] = useState(false);
+  const navigate = useNavigate();
+
+  const handleSelect = (i) => {
```

Comment:
suggestion :what is 'i' try to use some meaningful or specific name that could define its purpose.

Reply:
Sure